rollouts/docs/development.md

```diff
@@ -0,0 +1,84 @@
+# Development
+
+## Changing Rollouts API
+
+If you change the API resources, in `api/.../*.go`, update the generated code and manifests by running:
+
+```sh
+make generate
+make manifests
+```
+
+Remove old CRDs and apply the new CRDs to your cluster by running:
```

Comment:
hmm...Add a note for dropping existing Rollout objects ? (to be on the safer side) ?

Reply:
Done.